### ISSUE TITLE
azure-quickstart-templates/101-backup-oms-monitoring/ Issue

### ISSUE THREAD

**Issue by singh1982** [Azure/azure-quickstart-templates#5799]
--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

Sorry to hear you had a bad experience with one of the templates :worried: But, in case you're just asking a question, we're happy to help. You can also check if the question might already have been asked here https://github.com/Azure/azure-quickstart-templates/issues?utf8=%E2%9C%93&q=is%3Aissue

We've created an outline of recommended sections to fill out that will help make this Pull Request awesome!

--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

[Template Name goes here](Template link goes here)

### Issue Details
Getting below error when  open Solution from log Analytics

An error has occurred
{"error":{"message":"The request had some invalid properties","code":"BadArgumentError","innererror":{"code":"SyntaxError","message":"Syntax Error","information":{"details":"{\"error\":{\"code\":\"Bad request\",\"message\":\"Request is invalid and cannot be executed.\",\"innererror\":{\"code\":\"SEM0100\",\"message\":\"'project' operator: Failed to resolve scalar expression named 'AlertSeverity_s'\",\"@context\":{\"activityId\":\"f286c650-7cd6-455a-a706-287bd4cdb5e2\"},\"@errorCode\":\"SEM0100\",\"@errorMessage\":\"'project' operator: Failed to resolve scalar expression named 'AlertSeverity_s'\"}}}"}}}}
See less
a few seconds ago

### Repro steps (*if necessary, delete otherwise*)
1. Log Analytic --> Click on Workspace
2. Settings --> Solutions --> Azure Backup
3. Click on Azure Backup Monitoring Solution-- After Sometime Error will Come Notification Windows
4.
5.
![image](https://user-images.githubusercontent.com/47524790/52562558-e2c87880-2e24-11e9-9c8b-c298895e559e.png)

**Comment by MarkDNL** [comment]
*101-backup-oms-monitoring* https://github.com/Azure/azure-quickstart-templates/tree/master/101-backup-oms-monitoring

The error described above is caused by invalid queries in the `Alerts form Azure Resources Backup` and `Alerts from On-prem Backup` tiles in the solution. These queries throw the following syntax error:

`'where' operator: Failed to resolve column or scalar expression named 'AlertUniqueId_s'`

The underlying queries are for example as follows:

```kusto
AzureDiagnostics
| where Category == "AzureBackupReport"
| where OperationName == "Alert"
| where AlertUniqueId_s != ""
| summarize arg_max(TimeGenerated, *) by AlertUniqueId_s
| extend CurrentStatus = AlertStatus_s
| join
(
    AzureDiagnostics
    | where Category == "AzureBackupReport"
    | where OperationName == "Alert"
    | where AlertStatus_s == "Active"
    | summarize HitCount = count() by AlertUniqueId_s
)
on AlertUniqueId_s
| project AlertUniqueId_s, HitCount , CurrentStatus , AlertCode_s, AlertSeverity_s
| where CurrentStatus == "Active"
| summarize count() by AlertSeverity_s
```

Is this solution still maintained? It is not mentioned anymore at https://docs.microsoft.com/en-us/azure/backup/backup-azure-configure-reports

**Comment by pvrk** [comment]
Yes, this solution is still maintained and documented [here](https://docs.microsoft.com/azure/backup/backup-azure-monitoring-use-azuremonitor)
This error occurs when no alerts are pumped to the relevant workspace. This usually happens when the template is deployed before the diagnostic settings are configured. One can ignore this error because it doesn't appear once alerts are generated and directed to LA workspace.

**Comment by pvrk** [comment]
I will take an action item to modify the template such that it doesn't throw this error, probably using columnIfExists.

**Comment by MarkDNL** [comment]
Thank you. In this case all backups went fine, so there were no alerts.

**Comment by MarkDNL** [comment]
I tried to fix it using `columnIfExist` but didn't succeed yet.

**Comment by pvrk** [comment]
@MarkDNL : We have [revamped the template](https://docs.microsoft.com/azure/backup/backup-azure-monitoring-use-azuremonitor#deploy-a-solution-to-the-log-analytics-workspace) recently. Changed the view and updated it such that this problem will not arise. Please check and let us know.

**Comment by acapodilupo** [comment]
@pvrk it appears the URL used for the artifacts location for the linked templates is invalid. This is causing the deployment to always fail. Has this url been updated? 

https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-backup-la-reporting/

Thanks.

**Comment by pvrk** [comment]
#please-close 
Can reopen if the problem still persists

**Comment by vrmalush** [comment]
Issue still persists. If column is not present then search query in Azure monitor alerts shows an error 'Request has invalid inputs'.
